src/stream.ts

```diff
@@ -1,70 +1,140 @@
-import { Stream } from '@libp2p/interface-connection';
+import { Stream, Direction } from '@libp2p/interface-connection';
 import { StreamStat } from '@libp2p/interface-connection';
-import { logger } from '@libp2p/logger';
+// import { logger } from '@libp2p/logger';
```

Comment:
You don't need/want any logging?
I suppose that makes sense. The Stream's methods could be called very frequently.